src/patternfly/components/Alert/alert.scss

```
@@ -3,215 +3,171 @@
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-alert--grid-template-columns: max-content 1fr max-content;
```

```suggestion
  --pf-c-alert--GridTemplateColumns: max-content 1fr max-content;
```

Or `AlertColumnWidth`, `TitleColumnWidth`, `ActionColumnWidth`